### PR TITLE
Feature/andrewheyer/tss 3270

### PR DESCRIPTION
**Pull request by andrewbheyer**

### CODE REVIEW

File: tests/test_csc.py [hunk 1]
```diff
@@ -160,6 +160,107 @@ def test_standard_state_transitions(self):
...
+            cmd_attr = getattr(harness.remote, f"cmd_enterControl")
+            state_coro = harness.remote.evt_summaryState.next(timeout=1.)
+            id_ack = await cmd_attr.start(cmd_attr.DataType(), timeout=1.)
+            state = await state_coro
+            self.assertEqual(id_ack.ack.ack, harness.remote.salinfo.lib.SAL__CMD_COMPLETE)
+            self.assertEqual(id_ack.ack.error, 0)
+            self.assertEqual(harness.csc.summary_state, salobj.State.STANDBY)
+            self.assertEqual(state.summaryState, salobj.State.STANDBY)
+            # set to FAULT; new state is FAULT
+            harness.csc._summary_state = salobj.State.FAULT
```
Comment (tribeiro) [inline tests/test_csc.py]: Transitioning to a fault state and reporting the summary state can be achieved in a single line by using the `summary_state` method of the `BaseCsc` class. Any reason for doing it this way?

File: tests/test_csc.py [hunk 2]
```diff
@@ -160,6 +160,107 @@ def test_standard_state_transitions(self):
...
+            cmd_attr = getattr(harness.remote, f"cmd_start")
+            state_coro = harness.remote.evt_summaryState.next(timeout=1.)
+            id_ack = await cmd_attr.start(cmd_attr.DataType(), timeout=1.)
+            state = await state_coro
+            self.assertEqual(id_ack.ack.ack, harness.remote.salinfo.lib.SAL__CMD_COMPLETE)
+            self.assertEqual(id_ack.ack.error, 0)
+            self.assertEqual(harness.csc.summary_state, salobj.State.DISABLED)
+            self.assertEqual(state.summaryState, salobj.State.DISABLED)
+            # set to FAULT; new state is FAULT
+            harness.csc._summary_state = salobj.State.FAULT
```
Comment (tribeiro) [inline tests/test_csc.py]: Same as in [here](https://github.com/lsst-ts/ts_scheduler/pull/19/files#diff-d9365181a6fb138131218b600a0dfccfR191)

File: tests/test_csc.py [hunk 3]
```diff
@@ -160,6 +160,107 @@ def test_standard_state_transitions(self):
...
+            cmd_attr = getattr(harness.remote, f"cmd_enable")
+            state_coro = harness.remote.evt_summaryState.next(timeout=1.)
+            id_ack = await cmd_attr.start(cmd_attr.DataType(), timeout=1.)
+            state = await state_coro
+            self.assertEqual(id_ack.ack.ack, harness.remote.salinfo.lib.SAL__CMD_COMPLETE)
+            self.assertEqual(id_ack.ack.error, 0)
+            self.assertEqual(harness.csc.summary_state, salobj.State.ENABLED)
+            self.assertEqual(state.summaryState, salobj.State.ENABLED)
+            # set to FAULT; new state is FAULT
+            harness.csc._summary_state = salobj.State.FAULT
```
Comment (tribeiro) [inline tests/test_csc.py]: Same as in [here](https://github.com/lsst-ts/ts_scheduler/pull/19/files#diff-d9365181a6fb138131218b600a0dfccfR191)

File: tests/test_csc.py [hunk 4]
```diff
@@ -160,6 +160,107 @@ def test_standard_state_transitions(self):
         # with pytest.raises(AttributeError):
         #     m.previous_state = "ENABLED"
 
+    def test_every_fault_transition(self):
+        """Test standard CSC Fault transitions.
```
[the remark this answers is not in the document]
Comment (andrewbheyer) [reply tests/test_csc.py]: Thanks, I forgot to even look at the docstring on this PR.

File: tests/test_csc.py [hunk 5]
```diff
@@ -160,6 +160,107 @@ def test_standard_state_transitions(self):
...
+         Returns
+        -------
```
[the remark this answers is not in the document]
Comment (andrewbheyer) [reply tests/test_csc.py]: fixed.

**Comment by tribeiro** [comment]
I think this PR is no longer relevant. Closing it.

**Pull request closed without merging**